doc/api/deprecations.md

```diff
@@ -430,9 +430,12 @@ changes:
   - version: v7.0.0
     pr-url: https://github.com/nodejs/node/pull/8217
     description: Runtime deprecation.
+  - version: REPLACEME
+    pr-url: REPLACEME
+    descripion: Deprecation revoked
 -->
 
-Type: Runtime
+Type: Deprecation revoked
```

Comment:
Hmmm.. that's not an actual status we have. May need the TSC to discuss how we generally want to handle this kind of case so it's not just a one-off

Reply:
Yes, that's fine, but we still don't have an actual process for revoking a deprecation. That's what we should settle on. So far it's been done on a case by case basis.